tests/hardening/5.2.23_limit_ssh_max_sessions.sh

```diff
@@ -7,6 +7,13 @@ test_audit() {
     # shellcheck disable=2154
     run blank /opt/debian-cis/bin/hardening/"${script}".sh --audit-all
 
+    echo "maxsessions 15" >>/etc/ssh/sshd_config
+
+    describe Running too permissive
+    register_test retvalshouldbe 1
+    register_test contain "[ KO ] 15 is higher than recommended 10"
+    run permissive /opt/debian-cis/bin/hardening/"${script}".sh --audit-all
+
```

Comment:
Blocker : Please add a maxsessions lower than recommended value and check that the check passes.